gadgets/python/GadgetInstrumentationStreamController.cpp

```diff
@@ -98,6 +98,19 @@ namespace Gadgetron
     return GADGET_OK;
   }
 
+  int GadgetInstrumentationStreamController::return_recondata(ACE_Message_Block* mb){
+    GILLock lock;
+    auto m1 = AsContainerMessage<IsmrmrdReconData>(mb);
+    try{
+      python_gadget_.attr("put_next")(*m1->getObjectPtr());
+    } catch(boost::python::error_already_set const &) {
+      GERROR("Passing recondata on to python wrapper gadget failed\n");
+      PyErr_Print();
```

Comment:
I know I did this in the existing code too, but this should use the following function from `python_toolbox.h`: `std::string pyerr_to_string(void)` instead of just letting Python print to `stderr`.